src/main/webapp/js/userMapData.json

```diff
@@ -332,9 +360,12 @@
 ["Edinburgh Napier University","UK"],
 ["Emmanuel College","UK"],
 ["Home Learning College","UK"],
+["Hult International Business School","UK"],
 ["Imperial College London","UK"],
+["Kingston University London","UK"],
 ["Leeds Trinity University","UK"],
 ["London School of Economics","UK"],
+["London School of Economics and Political Science","UK"],
```

Comment:
These refer to the same institute, although `London School of Economics` seems to be a common way to refer to the institute. Kept `London School of Economics and Political Science` as based off their website.